cloudevents/extensions/jwt.md

```diff
@@ -0,0 +1,147 @@
+# JWT Extension
+
+This extension provides a way to map JWT (JSON Web Token) values in the formats of JWE
+ (JSON Web Encryption) or JWS (JSON Web Signature).
+ 
+JWE MAY be used to encrypt a CloudEvent data and JWS MAY be used to sign CloudEvent data
+
+This extension does not support [JWE Json Serialization][jwe-json-serialization] 
+
+When using [JWE Cipher Text][jwe-ciphertext] MUST be mapped onto `data` and
+ `datacontenttype` MUST be set to `application/octet-stream`
+
+When using JWS the signed data MUST be decoded from base64url format and mapped onto
+`data` in `Binary` format where `datacontenttype` is taken from the `cty` property
+of the JOSE header, if no `cty` property exists `datacontenttype` MUST be set to `application/octet-stream`
+  
+## Attributes
+
+### jose
+- Type: `Binary`
+- Description: utf-8 encoded JOSE Header value.
+    Both JWS and JWE have JOSE Header definitions.
+- Constraints:
+  - REQUIRED
+  - MUST be a non-empty octet sequence
+- Examples:
+  - `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9` (`{"alg": "HS256","typ": "JWT"}`)
+
+### jwssignature
+- Type: `Binary`
+- Description: JWS Signature.
+    Computed over the complete JOSE header and `data` value
+- Constraints:
+  - OPTIONAL
+  - MUST be a non-empty octet sequence
+- Examples:
+  - `SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c` 
+
+### jweencryptedkey
+- Type: `Binary`
+- Description : [JWE Encrypted Key][jwe-encrypted-key].
+    Encrypted Content Encryption Key value. Note that for some algorithms, 
```

Comment:
Can we make the first two "sentences" into readable sentences? :-) Same for previous property. Even if we just add "The" to the start of it, at least then it's a complete sentence.